### ISSUE TITLE
Bug: Purple shadow/outline on focused menu option item does not have enough color contrast to the menu option item border

### ISSUE THREAD

**Issue by longmatthewh** [dequelabs/cauldron#877]
### Steps to reproduce

From our a11y team:

```
As per new WCAG changes - When a focus indicator appears Inside the component, it needs to contrast with the adjacent color(s) within the component.

For submenu options, the focus indicator is present inside the component and is surrounded by  grey color (inside and outside) and the purple color is not replacing the border grey color.

We observed this when checking for color issues using color contrast analyzer tool.
```

![image (4)](https://user-images.githubusercontent.com/3958577/215843135-cf6bf0f0-e9ec-45b9-b9cf-9ab2d25cea16.png)

### Expected Behavior

The purple focus outline color `#C11DBE` should have enough contrast with colors on either side of it. It is recommend by @bobbyomari to add spacing of the same menu item background color (ex. light theme is `#f2f2f2`).

<img width="927" alt="Screen Shot 2023-01-31 at 9 08 42 AM" src="https://user-images.githubusercontent.com/3958577/215844126-e9868515-4093-4f79-9480-24ea18e75769.png">

### Actual Behavior

A bug happened!The purple focus outline color `#C11DBE` does not have enough contrast with the menu item border color `#CCCCCC`.

### Version

5.0.0

### What browsers are you experiencing the problem on?

Chrome

### Anything else we should know?

Additional information
